css/owned-list-pages.styl

```diff
@@ -30,4 +47,20 @@
     .active
       border: 2px solid
 
+  .card-search
```

Comment:
probably not used? Since owned card list is only for collections and favourites now? Possibly there should be no changes to this file now...